rmgpy/chemkin.pyx

```diff
@@ -1700,7 +1700,8 @@ def write_reaction_string(reaction, java_library=False):
 ################################################################################
 
 
-def write_kinetics_entry(reaction, species_list, verbose=True, java_library=False, commented=False):
+def write_kinetics_entry(reaction, species_list, verbose=True, java_library=False, commented=False,
+                         coverage_dependence=False):
```

Comment:
I would have thought if coverage dependence is turned off, we wouldn't have coverage dependent rates in the model, there would be no coverage dependent parameters, and so the chemkin writing of them wouldn't need to be disabled - they just wouldn't be there. Why do we need to put all this "don't write coverage dependent parameters" code into the Chemkin-writing machinery?

Reply:
I was having issues with it still writing them while turned off, but I guess I got a little carried away